packages/types/src/identity/index.ts

```diff
@@ -22,16 +22,24 @@ export interface INewIdentityRequest {
   messageSignature?: string;
 }
 
+export interface IImportIdentityArgs {
+  name: string;
+  trapdoor: string;
+  nullifier: string;
+  urlOrigin?: string;
+}
+
```

Comment:
I think we need a validation step before accepting `trapdoor` and `nullifier`.

Reply:
Validation will be done in https://github.com/CryptKeeperZK/crypt-keeper-extension/pull/993

Reply:
Validation is done in https://github.com/CryptKeeperZK/crypt-keeper-extension/pull/993